src/components/autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,161 @@
+@keyframes list-out {
+  0% {
+    animation-timing-function: linear;
+  }
+  50% {
+    opacity: 0;
+    height: 40px;
+    animation-timing-function: ease-in;
+  }
+  100% {
+    height: 0;
+    opacity: 0;
+  }
+}
+@keyframes list-in {
+  0% {
+    opacity: 0;
+    height: 0;
+    animation-timing-function: linear;
+  }
+  50% {
+    opacity: 1;
+    height: 40px;
+    animation-timing-function: ease-out;
+  }
+  100% {
+    opacity: 1;
+    height: 40px;
+  }
+}
+body {
+  max-width: 800px;
+  margin: 0 auto;
+  height: 100%;
+  md-content {
+    overflow: visible;
+  }
+  md-autocomplete {
+    background: white;
+    box-shadow: 0 2px 5px rgba(black, 0.25);
+    border-radius: 2px;
+    display: block;
+    height: 40px;
+    position: relative;
+    overflow: visible;
+
+    md-autocomplete-wrap {
+      display: block;
+      position: relative;
+      overflow: visible;
+      height: 40px;
+
+      md-progress-linear {
+        position: absolute;
+        bottom: 0; left: 0; width: 100%;
+        height: 3px;
+        transition: none;
+
+        .md-container {
+          transition: none;
+          top: auto;
+          height: 3px;
+        }
+        &.ng-enter {
+          transition: opacity 0.15s linear;
+          &.ng-enter-active {
+            opacity: 1;
+          }
+        }
+        &.ng-leave {
+          transition: opacity 0.15s linear;
+          &.ng-leave-active {
+            opacity: 0;
+          }
+        }
+      }
+    }
+    input {
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 100%;
+      box-sizing: border-box;
+      border: none;
+      box-shadow: none;
+      padding: 0 15px;
+      font: 400 14px/40px Roboto, sans-serif;
+      height: 40px;
+      outline: none;
+      z-index: 2;
+      background: transparent;
+    }
+    button {
```

Comment:
Agreed.  Right now, this button is somewhat placeholder - I was trying to wait until our icons were ready to do the PR because ideally, there should be an icon for this, but I wanted to get some eyes on it early because I anticipated having to make some more changes based on feedback.